.github/workflows/cli.yml

```diff
@@ -133,10 +125,7 @@ jobs:
         with:
           cdn: ${{ env.PROD_ASSETS }}
           assets_version: ${{ env.ASSETS_VERSION }}
-
-      - name: Install cargo-nextest
-        run: |
-          curl -LsSf https://get.nexte.st/latest/mac | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
+          platform: macos
```

Comment:
`cargo-nextest` isn't installed anymore, no?
And why did it work before without specifying the platform argument?